packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -253,6 +323,41 @@ class ContainerPlugin {
         );
       },
     );
+
+    // add include of federation runtime
+    compiler.hooks.thisCompilation.tap(
+      PLUGIN_NAME,
+      (compilation: Compilation, { normalModuleFactory }) => {
+        const federationRuntimeDependency =
+          federationRuntimePluginInstance.getDependency();
+
+        const logger = compilation.getLogger('ContainerPlugin');
+        const hooks = FederationModulesPlugin.getCompilationHooks(compilation);
+        compilation.dependencyFactories.set(
+          FederationRuntimeDependency,
+          normalModuleFactory,
+        );
+        compilation.dependencyTemplates.set(
+          FederationRuntimeDependency,
+          new ModuleDependency.Template(),
+        );
+
+        compilation.addInclude(
+          compiler.context,
+          federationRuntimeDependency,
+          { name: undefined },
```

Comment:
name: undefined will make it include it in all entry files

Reply:
this includes into container entry and into all entrypoints as well. But i do run theis hook again in federation runtime plugin since the build may not call container plugin if there is no exposes, so i still need the federation runtime to be added to entrypoints by someone